sdk/core/core-rest-pipeline/src/policies/tracingPolicy.ts

```diff
@@ -1,13 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { getTracer, getTraceParentHeader } from "@azure/core-tracing";
-import { SpanOptions, SpanKind } from "@opentelemetry/api";
+import {
+  getTraceParentHeader,
+  OperationTracingOptions,
+  createSpanFunction
+} from "@azure/core-tracing";
+import { SpanKind } from "@opentelemetry/api";
 import { PipelineResponse, PipelineRequest, SendRequest } from "../interfaces";
 import { PipelinePolicy } from "../pipeline";
 import { URL } from "../util/url";
 import { getUserAgentValue } from "../util/userAgent";
 
+const createSpan = createSpanFunction({
+  packagePrefix: undefined,
```

Comment:
If these can be undefined, should we make the property itself optional? Or maybe keep it as is, since this use case is so niche, we don't want folks to forget to set them in other packages?

Reply:
Yeah, that was my thought. I actually had them optional originally but thought it'd be too easy to not realize you should set them.